server/store.go

```diff
@@ -194,3 +196,25 @@ func (p *Plugin) removeSuperUserToken() error {
 
 	return nil
 }
+
+func (p *Plugin) setFollowStatusForUser(userID string, enabled bool) *model.AppError {
+	key := fmt.Sprintf("%v_%v", zoomFollowStatusKeyByUserId, userID)
+	return p.API.KVSet(key, []byte(strconv.FormatBool(enabled)))
+}
+
+func (p *Plugin) getFollowStatusForUser(userID string) (bool, *model.AppError) {
+	key := fmt.Sprintf("%v_%v", zoomFollowStatusKeyByUserId, userID)
```

Comment:
Can we create a field `Settings` on the `zoom.OAuthUserInfo` struct, which would then have a `FollowStatus` field? If we do this, we should probably rename the `zoom.OAuthUserInfo` struct to `zoom.UserInfo` as well.